rasa_sdk/endpoint.py

```diff
@@ -90,7 +92,13 @@ async def health(_) -> HTTPResponse:
     @app.post("/webhook")
     async def webhook(request: Request) -> HTTPResponse:
         """Webhook to retrieve action calls."""
-        action_call = request.json
+        if request.headers.get("Content-Encoding") == "deflate":
+            # Decompress the request data using zlib
+            decompressed_data = zlib.decompress(request.body)
```

Comment:
Shouldn't this be `request.json` instead of `request.body`?

Reply:
Compressed body will not be stored in json property.